### PR TITLE
DOM Creation | Jenny Tran

### PR DESCRIPTION
**Pull request by jennydtran**
Screen Shot 2020-10-21 at 4 05 50 PM
Screen Shot 2020-10-21 at 4 05 57 PM

### CODE REVIEW

File: dom-creation/main.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    name: 'Blastoise',
+    description: 'It crushes its foe under its heavy body to cause fainting. In a pinch, it will withdraw inside its shell.',
+    imageUrl: 'images/blastoise.png'
+  }
+];
+
+function renderPokemon(pokemon) {
+  var divColumnThird = document.createElement('div');
+  divColumnThird.setAttribute('class', 'column-third');
+  row.appendChild(divColumnThird);
```
Comment (codingcodymiller) [inline dom-creation/main.js:63]: You should not be appending this to the DOM inside of the `renderPokemon` function. The goal of this function is to create the DOM tree, and return that DOM tree by returning the element that holds all the other elements. You should be doing the actual appending with the return value of the function call in your loop 👍

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: dom-creation/main.js (hunk 1 above)
Comment (jennydtran) [reply dom-creation/main.js:63]: I knew that was where I went wrong and I couldn't explain it to Brett. I'll try going through the documentation again.

**Comment by jennydtran** [comment]
I know why everything went wrong now. I did not have a value returned from the renderPokemon function *slaps forehead*

**Review by codingcodymiller — APPROVED** [review]
🥳 🎉 🎉 🎉 🎉

**Pull request merged**